package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "razzle start",
     "build": "razzle build",
     "lint": "eslint --ext .js,jsx ./src",
-    "test": "razzle test --env=jsdom",
+    "test": "npm run lint & npm run test:unit",
```

Comment:
Use 2 ampersands rather than one, a single ampersand could be slightly confusing as it looks like it might run as a background process. 

After renamed above to `test:lint`, make that change here also.